tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -13,7 +13,8 @@ from (
     if(currentPmCounts.count_num > 1, NULL, currentPmCounts.owner) as owner
   from
     ContactDetails cd
-  left join GmcDetails gmc on (gmc.id = cd.id)
+  inner join GmcDetails gmc on (gmc.id = cd.id)
+  and not lower(gmc.gmcNumber) = "unknown"
```

Comment:
Does the inner join sort that?

Reply:
Currently, there are more null values than "unknown" and no blank/whitespace values.

They won't be filtered out by the inner join and there are a small number which appear to be for "CURRENT" trainees so they won't all be excluded because of that.